src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs

```diff
@@ -890,7 +890,7 @@ TimeSpan DeriveKeepAliveTimeout(IChannelHandlerContext context, ConnectPacket pa
         /// <param name="context"><see cref="IChannelHandlerContext" /> instance.</param>
         void CompleteConnect(IChannelHandlerContext context)
         {
-            CommonEventSource.Log.Info("Connection established.", this.ChannelId, this.Id);
+            CommonEventSource.Log.Info($"Connection established. Keep-alive timeout {this.keepAliveTimeout}.", this.ChannelId, this.Id);
```

Comment:
"... Keep-Alive timeout: {this.keepAliveTimeout}"

Reply:
I will modify the log string